.pre-commit-config.yaml

```diff
@@ -31,13 +31,6 @@ repos:
       - id: ruff
         args: [--fix, --exit-non-zero-on-fix]
 
-  # Deterministic python formatting:
-  - repo: https://github.com/psf/black
-    rev: 23.11.0
-    hooks:
-      - id: black
-        language_version: python3.11
-
```

Comment:
Did you also intend to add the `ruff` formatter in the `pre-commit` hook above, and remove the black dependency and configuration from `pyproject.toml` and add the ruff formatting configuration?

Reply:
I believe ruff was already added, so I was removing black. I removed the dependency though, and slightly changed the ruff configuration to exactly match PUDL. Is there anything else we need to do? It seems to be handling formatting for me just fine.

Reply:
`ruff` the linter was added, but there's a separate `ruff format` command, and it needs to be added to the pre-commit config, which it looks like you did.

It looks like there's still `black` configuration information in the `pyproject.toml`, and the `tool.ruff.format` configuration should be brought over from PUDL.